src/keycloak/admin/users.py

```diff
@@ -72,26 +89,115 @@ def by_id(self, user_id):
 
 
 class User(KeycloakAdminBase):
+    _BASE = "/auth/admin/realms/{realm}/users/{user_id}"
     _paths = {
-        'single': '/auth/admin/realms/{realm}/users/{user_id}'
+        'single': _BASE,
+        'group': _BASE + '/groups',
```

Comment:
Can you split out the groups related methods into a `UserGroups` class? Just like the ClientRoles? That's more in line with the architecture of the rest of the client. If done you can call the functionality in the following way:
```
client = KeycloakAdmin(*kwargs)
client.realms.by_name('my-realm').users.by_id(123).groups.all()
client.realms.by_name('my-realm').users.by_id(123).groups.add(group_id)
client.realms.by_name('my-realm').users.by_id(123).groups.delete(group_id)
```